themes/openy_themes/openy_carnation/src/scss/component/_addtocalendar.scss

```diff
@@ -77,7 +77,7 @@
 /* Blue */
 .atc-style-blue .atcb-link:before {
   margin-right: 8px;
-  font-family: FontAwesome;
+  font-family: "Font Awesome 5 Free";
```

Comment:
could you make this a variable in _variables.scss for future possible changes?

Reply:
@podarok done